presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergPageSink.java

```diff
@@ -376,6 +378,23 @@ public static Object getIcebergValue(Block block, int position, Type type)
         throw new UnsupportedOperationException("Type not supported as partition column: " + type.getDisplayName());
     }
 
+    private static Object adjustForTransformCalculationIfNecessary(SqlFunctionProperties functionProperties, Type type, Object value)
```

Comment:
nit: Should we rename this to something like `adjustTimestampForPartitionTransform`? Just to make it clearer that this is for timestamp-based transforms

Reply:
OK, fixed!